packages/babel-parser/test/fixtures/typescript/types/typeof-private/output.json

```diff
@@ -1,6 +1,9 @@
 {
   "type": "File",
   "start":0,"end":379,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":23,"column":1,"index":379}},
+  "errors": [
+    "SyntaxError: Private name #a is not defined. (11:18)"
```

Comment:
Can you extract it into a new test case? Mixing valid case with invalid case would be confusing.

Reply:
test cases updated.